graylog2-web-interface/src/components/search/MessageFieldSearchActions.jsx

```diff
@@ -23,6 +23,18 @@ const MessageFieldSearchActions = React.createClass({
     );
   },
   render() {
+    let messageField = this.props.message.fields[this.props.fieldName];
+    let extractors;
+    if(typeof messageField === 'string') {
+      extractors = (<li className="dropdown-submenu left-submenu">
+        <a href="#">Create extractor for field {this.props.fieldName}</a>
+        <ul className="dropdown-menu">
+            {ExtractorUtils.EXTRACTOR_TYPES.map(extractorType => this._formatExtractorMenuItem(extractorType))}
+        </ul>
+      </li>);
+    } else {
+        extractors = (<MenuItem disabled={true}>Extractors can only be used with string fields</MenuItem>);
```

Comment:
Same as in the other file: 

- `key` can be a regular string
- We don't need to pass the boolean value to the `disabled` flag (it breaks the [react/jsx-boolean-value](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md) rule)